### ISSUE TITLE
Create game board with basic styling

### ISSUE THREAD

**Issue by cmccormack** [cmccormack/minesweeper#6]
Game board can be any size but we should try to limit it for now for testing.  We can add additional sizes in the form of a dropdown or inputs as well

**Comment by bradtaniguchi** [comment]
According to the minesweeper wiki, the game board had the following mines and dimensions:
- Beginner: 8x8 10 mines
- Intermediate 16x16 40 mines
- Expert 24x24 99 mines

I think we can start with one of these presets, then add support for the other ones, or just jump to custom settings.
[ref](http://www.minesweeper.info/wiki/Windows_Minesweeper)

**Comment by cmccormack** [comment]
@bradtaniguchi Yeah I think presets are nice to have but we should have the functionality to build any size we want dynamically.
